src/registrar/templatetags/custom_filters.py

```diff
@@ -48,3 +49,17 @@ def contains_checkbox(html_list):
         if re.search(r'<input[^>]*type="checkbox"', html_string):
             return True
     return False
+
+
+@register.filter
+def get_organization_long_name(organization_type):
+    organization_choices_dict = {}
+
+    for name, value in DomainApplication.OrganizationChoicesVerbose.choices:
```

Comment:
**comment (non-blocking)**: This is idiomatically `dict(DomainApplication.OrganizationChoicesVerbose.choices)`. e.g. https://gist.github.com/OmenApps/3eef60ba4204f3d1842d9d7477efcce1#file-django_choices-txt-L28